<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialAssistant</class>
 <widget class="QWidget" name="SerialAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Window</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string> 串口设置</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>串口号</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_portName"/>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_portBaud"/>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_portDatabit"/>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_portStopbit"/>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_portParity"/>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButton_OpenPort">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ScanPort">
       <property name="text">
        <string> 扫描</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit_recive">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>691</width>
     <height>391</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::WidgetWidth</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>741</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_TextBox">
    <attribute name="title">
     <string>单条发送</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0" colspan="2">
      <widget class="QTextEdit" name="textEdit_Input"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>多条发送</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QSplitter" name="splitter_22">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>651</width>
           <height>131</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter_20">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_1"/>
           <widget class="QPushButton" name="pushButton_1">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_2"/>
           <widget class="QPushButton" name="pushButton_4">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_3"/>
           <widget class="QPushButton" name="pushButton_2">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_4"/>
           <widget class="QPushButton" name="pushButton_3">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_5"/>
           <widget class="QPushButton" name="pushButton_5">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_21">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_6"/>
           <widget class="QPushButton" name="pushButton_6">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_7"/>
           <widget class="QPushButton" name="pushButton_7">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_17">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_8"/>
           <widget class="QPushButton" name="pushButton_8">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_18">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_9"/>
           <widget class="QPushButton" name="pushButton_9">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_19">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="lineEdit_10"/>
           <widget class="QPushButton" name="pushButton_10">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_23">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>150</y>
           <width>694</width>
           <height>23</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="pushButton_16">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_13">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>/page</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_14">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_15">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_Protocol">
    <attribute name="title">
     <string>协议传输</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>260</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="title">
    <string>接收区设置</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Recv_DisplayHEX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>43</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Recv_DispalyASCII">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearRecv">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>清除接收</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LogMode">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>日志模式</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_StopRecv">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>停止接收</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>490</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string> 发送区设置</string>
   </property>
   <widget class="QSplitter" name="splitter_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>28</y>
      <width>94</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QRadioButton" name="radioButton_InputHEX">
     <property name="text">
      <string>HEX</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_InputASCII">
     <property name="text">
      <string>ASCII</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>清除发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string> 发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tx_rx_number">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>label_tx_rx_number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
